src/main/java/michael/Ui.java

```diff
@@ -0,0 +1,24 @@
+package michael;
+
+public class Ui {
+    private static final String BORDER = "--------------------------------------";
+
+    public Ui() {
+        // Greet user first
+        printer("Hello! I'm Michael.\n" + "What can I do for you?");
+    }
+
+    public void showLoadingError(String error) {
+        System.out.println(error);
+    }
+
+    public void feedback(String feedback) {
```

Comment:
Would it be better to make this method name a verb?
```suggestion
    public void giveFeedback(String feedback) {
```